modules/serverless-apps-intro.adoc

```diff
@@ -0,0 +1,39 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative_serving/serverless-knative-serving.adoc
+// serverless/serving-creating-managing-apps.adoc
+
+[id="serverless-about-services_{context}"]
+= Serverless applications using Knative services
+
+To deploy a serverless application using {ServerlessProductName}, you must create a _Knative service_.
+Knative services are Kubernetes services, defined by a route and a configuration, and contained in a `.yaml` file.
+
+.Example Knative service YAML
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1beta1
+kind: Service
+metadata:
+  name: helloworld-go <1>
+  namespace: default <2>
+spec:
+  template:
+    spec:
+      containers:
+        - image: gcr.io/knative-samples/helloworld-go <3>
+          env:
+            - name: TARGET <4>
+              value: "Go Sample v1"
+----
+
+<1> The name of the application
+<2> The namespace the application will use
+<3> The URL to the image of the application
```

Comment:
image names are technically not URLs, as they doesn't start with `scheme:` (see https://tools.ietf.org/html/rfc3986#section-1.1.1 )